dotnet/src/IntegrationTests/Connectors/Memory/Postgres/PostgresMemoryStoreTests.cs

```diff
@@ -231,7 +231,7 @@ public async Task UpsertReplacesExistingRecordWithSameIdAsync()
         Assert.NotNull(actual);
         Assert.Equal(testRecord.Metadata.Id, key);
         Assert.Equal(testRecord2.Metadata.Id, actual.Key);
-        Assert.True(testRecord.Embedding.Span.SequenceEqual(actual.Embedding.Span));
+        Assert.False(testRecord.Embedding.Span.SequenceEqual(actual.Embedding.Span));
```

Comment:
It's interesting that previously we checked that `testRecord` embeddings should be equal to `actual`. But it looks like they should be different based on test scenario, so this change is valid.

Reply:
Yes, it originally verified that the embeddings value had been updated after the records with the same key were updated. However, when the embedded types are updated, the is updated to verity equal.